core/src/main/java/com/graphhopper/routing/util/parsers/AbstractAccessParser.java

```diff
@@ -79,15 +79,13 @@ public ConditionalTagInspector getConditionalTagInspector() {
      * Updates the given edge flags based on node tags
      */
     protected void handleNodeTags(IntsRef edgeFlags, Map<String, Object> nodeTags) {
-        if (!nodeTags.isEmpty()) {
-            // for now we just create a dummy reader node, because our encoders do not make use of the coordinates anyway
```

Comment:
Let's rename handleNodeTags? It sounds like something general, but cit is only called for edges tagged as `gh:barrier_edge`, and the same for the JavaDoc comment.